backend/src/main/java/dev/tripdraw/presentation/controller/PostController.java

```diff
@@ -48,12 +54,26 @@ public ResponseEntity<PostCreateResponse> createOfCurrentLocation(
             responseCode = "201",
             description = "사용자가 선택한 위치에 대한 감상 생성 성공."
     )
-    @PostMapping("/posts")
+    @PostMapping
     public ResponseEntity<PostCreateResponse> create(
             @Auth LoginUser loginUser,
             @Valid @RequestBody PostRequest postRequest
     ) {
         PostCreateResponse response = postService.addAtExistingLocation(loginUser, postRequest);
         return ResponseEntity.status(CREATED).body(response);
     }
+
+    @Operation(summary = "특정 감상 상세 조회 API", description = "특정한 1개 감상의 상세 정보 조회")
+    @ApiResponse(
+            responseCode = "200",
+            description = "특정 감상 상세 조회 성공."
+    )
+    @GetMapping("/{postId}")
+    public ResponseEntity<PostResponse> read(
+            @Auth LoginUser loginUser,
+            @PathVariable Long postId
+    ) {
+        PostResponse response = postService.read(loginUser, postId);
+        return ResponseEntity.status(OK).body(response);
+    }
```

Comment:
```suggestion
    public ResponseEntity<PostResponse> read(
            @Auth LoginUser loginUser,
            @PathVariable Long postId
    ) {
        PostResponse response = postService.read(loginUser, postId);
        return ResponseEntity.ok(response);
    }
```
메서드 만들어져 있는 거 쓰면 편할 것 같습니다!

Reply:
다른 컨트롤러 메서드도, 해당 메서드 사용하고 있어서
저도 사용했습니다! 감사합니다~